app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -91,6 +85,13 @@
 import java.util.List;
 import java.util.Locale;
 
+import mozilla.components.tabs.Tab;
+import mozilla.components.tabs.TabModel;
+import mozilla.components.tabs.TabView;
+import mozilla.components.tabs.TabViewProvider;
+import mozilla.components.tabs.TabsSession;
+import mozilla.components.tabs.TabsSessionProvider;
```

Comment:
I recommend to not overload the `mozilla.components.*` package and leave this namespace to code that is actually part of a component. If this should end up in a component eventually then maybe use something like `incubator.mozilla.components.*` until it's moved into a component.

Reply:
I relocate the implementations to `org.mozilla.rocket.tabs`